tests/test_mab.py

```
@@ -208,39 +331,31 @@ def test_mab_model_post_init_default_action_validation():
         DummyMab(actions=actions, strategy=ClassicBandit(), epsilon=None, default_action="action1")
 
 
-def test_mab_model_post_init_invalid_default_action(epsilon=0.1):
+def test_mab_model_post_init_invalid_default_action():
     """Test model_post_init validation for invalid default action."""
     actions = {"action1": Beta(), "action2": Beta()}
 
-    # Test case: default_action is not in the actions set - should raise AttributeError
     with pytest.raises(AttributeError, match="The default action must be valid action defined in the actions set."):
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)